### PR TITLE
Prevent initialize call on destroyed player ref

### LINKED ISSUE #137: Check for valid player instance before calling initialize
https://github.com/micnews/react-jw-player/blob/master/src/react-jw-player.jsx#L87

I receive many errors like:

```
TypeError: n.setup is not a function
```

Traced it to ../node_modules/react-jw-player/dist/helpers/initialize.js in default at line 26:1
The player instance is not guaranteed to be still available on the page at the time that initialize is called. We need to check to see if it is null since it may have been destroyed.

### PR DESCRIPTION
**Pull request by esetnik**
Fixes #137

### CODE REVIEW

**Comment by talkshoplivedev** [comment]
@danmakenoise @ellell is there a chance of this being merged soon?

**Comment by esetnik** [comment]
@talkshoplivedev i'm afraid the project might be abandoned. There isn't any activity from maintainers in about a year.

**Comment by talkshoplivedev** [comment]
@esetnik I just emailed the dev team at Mic for a definitive answer.

**Comment by esetnik** [comment]
I'd be happy to help out maintaining the project if they want to accept collaborators.

**Comment by talkshoplivedev** [comment]
@esetnik as would we.

**Comment by danmakenoise** [comment]
@esetnik @talkshoplivedev Hey! This repo is definitely no longer maintained as it stands, since the dev team that created this no longer exists. 

I'll need to see what permissions I still have, but I'll try to get this moved over to my personal account this weekend. It's not a good idea to keep working on it here, as none of the contributors have control over the `micnews` group, and it could be deleted at any time. I'll do some PR grooming first, and get it set up in another location.

Also, happy to add contributors once I get everything transfered. I no longer get daily use with JW Player in my current position, and could use some devs who are more active with it to help on that side.

**Comment by danmakenoise** [comment]
@esetnik this looks good, I'll merge it and do a publish.

**Comment by danmakenoise** [comment]
@esetnik @talkshoplivedev published `1.19.1`

**Comment by talkshoplivedev** [comment]
@danmakenoise thanks for the update. We at talkshop.live would be more than happy to help maintain and contribute. We had actually taken the initial steps to setup a separate repo, but would defer to you if you set it up and gave us permissions. There’s so much opportunity to improve on what has been built and we think it could really benefit others. Thanks again!

**Comment by danmakenoise** [comment]
@talkshoplivedev if you guys are willing to take on maintainer-ship, I am happy to update the readme to link to your new repo. I will ask that you guys create a new NPM package though. Let me know and I'll make sure the README gets updated on here and NPM.

**Comment by talkshoplivedev** [comment]
@danmakenoise I’ll connect with you separately on a few quick questions I have, but we should be able to have up on NPM by Wednesday.

**Comment by esetnik** [comment]
Did you consider reaching out to jwplayer org and see if they want to take over the project? They will have the most resources to maintain it and it’d be in their interest to take ownership I think.

**Comment by talkshoplivedev** [comment]
@esetnik for sure, and I have and will again, but they have been, let’s just say, less than supportive of this project when it comes to support. It’d make sense to get an official pass or accept from them, as you suggest.

**Comment by danmakenoise** [comment]
@esetnik @talkshoplivedev that's actually a good idea before we do this. I went ahead and flagged them down here https://github.com/jwplayer/jwplayer/issues/3591 We can see what they say.

File: src/react-jw-player.jsx [hunk 1]
```diff
@@ -82,6 +82,11 @@ class ReactJWPlayer extends Component {
 
     const component = this;
     const player = window.jwplayer(this.videoRef);
+    if (!player) {
```
Comment (raffij) [inline src/react-jw-player.jsx:85]: This won't actually work as that'll return the following:

`window.jwplayer("random")` => `{registerPlugin: ƒ}`

**Pull request merged**